api/desecapi/tests/test_rrsets.py

```diff
@@ -1636,3 +1638,131 @@ def test_create_my_rr_sets_ip_block(self):
         )
         self.assertStatus(response, status.HTTP_400_BAD_REQUEST)
         self.assertIn("IP address 3.2.2.5 not allowed.", str(response.data))
+
+    def test_create_ns_rrset(self):
```

Comment:
I've used delegation of dedyn subdomains myself and found it pretty useful for testing purposes, e.g. for the pqc testbench. Do we have to disallow it?

Reply:
The load problems last year stemmed from people trying to delegate their domains to Cloudflare, at the apex. It is my suspicion that if we only disallow *that*, we'll again soon have tens of thousands of registrations that then delegate a subdomain to Cloudflare. We're not prepared to handle these numbers, and I'm worried about that / would like to put stability first.

There are currently only very few lower-level delegations under LPS domains, and those continue to work. If it's ever needed, we can (1) do it in the backend (the model allows it, just not the REST endpoint), (2) add a flag that would allow certain accounts, or (3) use a custom domain.

What do you think?

Reply:
Resuming operation for a few (or even most) users is better than resuming for none - so let's go for it the way it is first.

For the record, another idea could be to somehow (2b) privilege accounts that have at least one delegated domain - but extend that privilege to all domains associated with that account.